maps.py

```diff
@@ -0,0 +1,16 @@
+"""
+Main file for launching Flask application.
```

Comment:
If we are moving templates & static into the module, this goes as well. It's required to run the module, and not customized by the user, so if templates and static go in (required to run, not customized), maps.py should as well. It's not like the requests module also has the main file to run it outside of the module.

Reply:
If not, I'll live, but that's just my opinion